src/frontend/src/pages/ProjectsPage/ProjectsTable.tsx

```diff
@@ -221,12 +191,7 @@ const ProjectsTable: React.FC = () => {
             );
           }
         }}
-        componentsProps={{
-          toolbar: {
-            showQuickFilter: true,
-            quickFilterProps: { debounceMs: 500 }
-          }
-        }}
+        componentsProps={{ toolbar: { showQuickFilter: true, quickFilterProps: { debounceMs: 500 } } }}
```

Comment:
Spaces between the } is pretty strange. It'd be better if it didn't make a bunch of auto-format changes. If you restart VS Code and change this (like make a new line) does it still go back to this?